test/objectwrap.cc

```diff
@@ -24,6 +24,11 @@ class Test : public Napi::ObjectWrap<Test> {
 public:
   Test(const Napi::CallbackInfo& info) :
     Napi::ObjectWrap<Test>(info) {
+    
+    if(info.Length() > 0){
```

Comment:
Nit: Please add a space between ) and { at this location and a few others below.

Reply:
Wen through and searched for /\){/ and took care of the rest.  I also noticed that I was using "finalize" and "finalizer" interchangeably in names of vars in the test.   Since the name of the new method is "Finalize", i updated any of the test vars with "finalizer" in the name to "finalize".